### PR TITLE
Fix/function return object

### PR DESCRIPTION
**Pull request by WFrancois**
Hey !

Quick fix to fix the export json : some functions now returns array :/

I fix in GetLFDStatus where the same function were used.

For example : https://github.com/tomrus88/BlizzardInterfaceCode/blob/master/Interface/FrameXML/LFGList.lua#L986

### CODE REVIEW

File: exportjson.lua [hunk 1]
```diff
@@ -172,7 +172,10 @@ do
 				end
 			end
 		end
-		local _, activityID = C_LFGList.GetActiveEntryInfo()
+		local activityEntryInfo = C_LFGList.GetActiveEntryInfo()
+
+		local activityID = activityEntryInfo['activityID']
```
Comment (jahraphael) [inline exportjson.lua]: could just do activityEntryInfo.activityID here right?

File: exportjson.lua [hunk 2]
```diff
@@ -172,7 +172,10 @@ do
 				end
 			end
 		end
-		local _, activityID = C_LFGList.GetActiveEntryInfo()
+		local activityEntryInfo = C_LFGList.GetActiveEntryInfo()
```
Comment (jahraphael) [inline exportjson.lua]: Do we need to guard against activityEntryInfo being nil here?

**Comment by WFrancois** [comment]
Hm, I think so yeah :p I'm used to php :p

Do you prefer like this ?

________________________________

File: exportjson.lua (hunk 1 above)
Comment (jahraphael) [inline exportjson.lua]: could just do activityEntryInfo.activityID here right? I think we index into tables like that more consistently in this code?

File: exportjson.lua [hunk 3]
```diff
@@ -181,7 +184,9 @@ do
 			if numActiveApps > 0 then
 				local applicants = C_LFGList.GetApplicants()
 				for j = 1, #applicants do
-					local id, status, pendingStatus, numMembers = C_LFGList.GetApplicantInfo(applicants[j])
+					local applicantInfo = C_LFGList.GetApplicantInfo(applicants[j])
```
Comment (jahraphael) [inline exportjson.lua]: Do we need to guard against applicantInfo being nil here?

File: core.lua [hunk 4]
```diff
@@ -400,7 +400,12 @@ do
 	-- returns the LFD status (returns the info based on what we are hosting a group for, or what we queued up for)
 	function GetLFDStatus()
 		-- hosting a keystone group
-		local _, activityID, _, _, name, comment = C_LFGList.GetActiveEntryInfo()
+		local activityInfo = C_LFGList.GetActiveEntryInfo()
```
Comment (jahraphael) [inline core.lua]: Do we need to guard against activityInfo being nil here?

File: core.lua [hunk 5]
```diff
@@ -400,7 +400,12 @@ do
...
+
+		local activityID = activtyInfo['activityID']
```
Comment (jahraphael) [inline core.lua]: Probably index the tables with dot operator vs array?

File: core.lua [hunk 6]
```diff
@@ -415,7 +420,13 @@ do
 		local j = 1
 		for i = 1, #applications do
 			local resultID = applications[i]
-			local _, activityID, name, comment, _, _, _, _, _, _, _, isDelisted = C_LFGList.GetSearchResultInfo(resultID)
+			local searchResultInfo = C_LFGList.GetSearchResultInfo(resultID)
+
+			local activityID = searchResultInfo['activityID']
```
Comment (jahraphael) [inline core.lua]: Index via dot operator instead of array

File: core.lua [hunk 7]
```diff
@@ -415,7 +420,13 @@ do
 		local j = 1
 		for i = 1, #applications do
 			local resultID = applications[i]
-			local _, activityID, name, comment, _, _, _, _, _, _, _, isDelisted = C_LFGList.GetSearchResultInfo(resultID)
+			local searchResultInfo = C_LFGList.GetSearchResultInfo(resultID)
```
Comment (jahraphael) [inline core.lua]: Do we need to guard against this being nil?

File: exportjson.lua (hunk 2 above)
Comment (WFrancois) [reply exportjson.lua]: Yep, shouldn't happen, but it might avoid evoid an error in edge case

File: core.lua (hunk 4 above)
Comment (WFrancois) [reply core.lua]: Done 👌

**Comment by WFrancois** [comment]
Everything should be changed 👌

**Comment by jahraphael** [comment]
Thanks Isak!

**Pull request merged**